packages/storefront-events-sdk/src/types/schemas/product.ts

```diff
@@ -2,7 +2,7 @@ export type Product = {
     productId: number;
     name: string;
     sku: string;
-    topLevelSku?: string | null;
+    topLevelSku?: string | null; // This is required in most cases and will be mandatory in future releases.
```

Comment:
Can we document the difference here or in the [example code](https://github.com/adobe/commerce-events/blob/main/examples/events/example-contexts/mock-product-context.md) differentiating between `sku` and `topLevelSku`

Reply:
Will be added here https://github.com/adobe/commerce-events/pull/172